util/fixtures/makeWorkorder.js

```diff
@@ -15,6 +15,7 @@ module.exports = function makeWorkorder(userId, workflowId) {
     finishDate: "2029-02-23T00:00:00Z",
     finishTime: "2029-02-23T00:00:00Z",
     summary: "Do the work",
+    // TODO: for some reason 'startTimestamp' now shows up as 'finish date/time' in the portal
```

Comment:
I guess there's no harm in setting it forward. Easy to revert when the weirdness around the sematincs of these fields is resolved and eventually validation is implemented.